src/__tests__/__snapshots__/downshift.aria.js.snap

```diff
@@ -0,0 +1,107 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`basic snapshot 1`] = `
+<div
+  aria-expanded="false"
+  aria-haspopup="listbox"
+  aria-labelledby="downshift-0-label"
+  aria-owns="downshift-0-menu"
```

Comment:
The `aria-owns="downshift-0-menu"` attribute is throwing an error on the Chrome axe-core plugin because it's referencing the unordered list. I have experimented with this a little; if you can append aria-owns when `aria-expanded` state is true, the error will go away. It's also okay to have `aria-owns` removed when Downshift is collapsed.

There's support for this idea in the [Rich Internet Applications combobox spec](https://www.w3.org/TR/wai-aria-1.1/#combobox), version 1.1:

> When a combobox is expanded, authors MUST ensure it contains or **owns** an element that has a role of listbox, tree, grid, or dialog. This element is the combobox popup. When the combobox is expanded, authors MUST set aria-controls on the textbox element to a value that refers to the combobox popup element.

Reply:
`aria-owns` is redundant here. Owns is an explicit way to say soemthing is a child of this element, without is having to be in the DOM. Your `listbox` is a direct DOM child so you can remove it.

Reply:
That's helpful! Is it harmful to include? Some use cases will have the listbox NOT be a direct DOM child. It'd be easier to just include it if it's not harmful?

Reply:
Absolutely, no harm is done when you add it regardless of DOM tree (as long as it matches the ID correctly of course). As you point out, super useful when using a positioning lib.